### PR TITLE
[EPDev] add occupancy selection

### PR DESCRIPTION
**Pull request by jikim1290**
add occupancy selection

### CODE REVIEW

**Comment by jikim1290** [comment]
Hi @ddobrigk @strogolo , can you please review and approve this PR to study occupancy dependence of the event plane? thanks!

File: Common/Tasks/qVectorsCorrection.cxx [hunk 1]
```diff
@@ -79,6 +79,9 @@ struct qVectorsCorrection {
   Configurable<float> cfgMaxDCArToPVcut{"cfgMaxDCArToPVcut", 0.1, "Maximum transverse DCA"};
   Configurable<float> cfgMaxDCAzToPVcut{"cfgMaxDCAzToPVcut", 1.0, "Maximum longitudinal DCA"};
 
+  Configurable<int> cfgMaxOccupancy{"cfgMaxOccupancy", 999999, "maximum occupancy of tracks in neighbouring collisions in a given time range"};
```
Comment (strogolo) [inline Common/Tasks/qVectorsCorrection.cxx:82]: @jikim1290 is the default value large enough not to affect the event selections? I think so, but I would like to have your confirmation

Comment (jikim1290) [reply Common/Tasks/qVectorsCorrection.cxx:82]: Yes it is

Comment (strogolo) [reply Common/Tasks/qVectorsCorrection.cxx:82]: Perfect, so from my side everything is fine and we can proceed with the approval @ddobrigk

**Review by ddobrigk — APPROVED** [review]

**Pull request merged**